docs/docs/deploy/flightcontrol.md

```diff
@@ -9,13 +9,92 @@ description: How to deploy a Redwood app to AWS via Flightcontrol
 ## Flightcontrol Deploy Setup
 
 1. In your project, run the command `yarn rw setup deploy flightcontrol --database=YOUR_DB_TYPE` where YOUR_DB_TYPE is `mysql` or `postgresql`
-2. Commit the changes and push to github.
-3. If you don't have an account, sign up at [app.flightcontrol.dev/signup](https://app.flightcontrol.dev/signup?ref=redwood).
-4. Create a new project.
-   1. Connect your GitHub account and select your repo.
-   2. Click the Redwood preset
-   3. Click "Create project" (do not add services to the UI during this step, the flightcontrol.json you added will be used for service config)
-5. After project is created, add your env vars under Environment Settings.
-   1. If using dbAuth, add the session secret key env variable in the Flightcontrol dashboard.
+2. NOTE: The RedwoodJS Flightcontrol sets up an array of environments and defaults you to the 1st one: 'Development', add to the array to deploy more environments.
```

Comment:
```suggestion
   - This will create a `flightcontrol.json` file with an array of environments. It defaults with a single Development environment. Add to the array to deploy more environments.
```